src/Generator/MethodGenerator.php

```diff
@@ -170,6 +172,8 @@ public static function fromArray(array $array)
                 case 'returntype':
                     $method->setReturnType($value);
                     break;
+                case 'returnsreference':
+                    $method->setReturnsReference($value);
```

Comment:
I think we need a `(bool)` cast here.

Alternatively, we can declare `@param array<...>`  with more specific types, but that will likely open another rabbit hole

Reply:
How about the existing methods, like setBody, setVisibility (string) or setAbstract, setFinal, setInterface, setStatic (bools)? Should those get relevant type casts as well?